extensions/amp-font/0.1/amp-font.js

```diff
@@ -124,7 +128,7 @@ export class AmpFont extends AMP.BaseElement {
       this.onFontLoadSuccess_();
     }).catch(error => {
       this.onFontLoadError_();
-      throw error;
+      log.warn(TAG_, error, this.element);
```

Comment:
@cramforce using console.log/warn, i have a branch that separates amp-dev logs vs publisher-dev logs, so this makes it easier for me to follow up. (where console.log/warn will always be publisher-dev logs)